src/Components/index.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+import '../Styles/styles.css';
```

Comment:
Writing all of the styles in one css file is wrong. Each compoennt should have its own.